src/main/kotlin/no/nav/familie/ef/søknad/mapper/kontrakt/BosituasjonMapper.kt

```diff
@@ -25,10 +25,6 @@ object BosituasjonMapper : MapperMedVedlegg<Bosituasjon, KontraktBosituasjon>(Bo
 
     private fun mapSamboer(bosituasjon: Bosituasjon): Søknadsfelt<PersonMinimum>? {
 
-        if (bosituasjon.skalGifteSegEllerBliSamboer?.verdi == true) {
```

Comment:
@ma10s Vi er litt usikkre på denne, den ble lagt til her: 96b91a0

Reply:
Jeg tar en titt

Reply:
Dette var løsningen på vi la inn for å håndtere problemene vi fikk med delt modell i UI på kort sikt. 

Her er commit melding: 
Bosituasjon gjenbrukes i framtidsplaner fix (#255)  
Problem: 
Gjenbruk av bosituasjon.samboerdetaljer i UI/modell 

Løsning: 
1. Hvis en søker har planer om å gifte seg i framtiden - ikke map samboerdetaler under bosituasjon 
2. Hvis en bruker har planer om å gifte seg i framtiden -> map bosituasjon.samboerdetaljer til Sivilstandsplaner

Reply:
Kan det være lurt å la den ligge i en overgangsperiode?